bundles/action/org.openhab.action.pushover/src/main/java/org/openhab/action/pushover/internal/Pushover.java

```diff
@@ -468,6 +486,13 @@ private static String pushover0(String apiKey, String user, String message, Stri
                             attachment, e.getMessage());
                 }
             }
+
+            if (htmlFormatting) {
+                parts.add(new StringPart(MESSAGE_KEY_HTML, MESSAGE_VALUE_HTML, UTF_8_ENCODING));
```

Comment:
Please use `StandardCharsets.UTF_8.name()`. Would you be so kind to replace it everywhere in this file?

Reply:
changed it